src/renderers/webgl/WebGLExtensions.js

```diff
@@ -54,6 +54,7 @@ function WebGLExtensions( gl ) {
 			if ( capabilities.isWebGL2 ) {
 
 				getExtension( 'EXT_color_buffer_float' );
+				getExtension( 'WEBGL_clip_cull_distance' );
```

Comment:
Optional extensions are queried at the startup of the renderer to check if they are available. I've added `WEBGL_clip_cull_distance` to the WebGL 2 section.